lib/Fmt_ast.ml

```diff
@@ -2376,6 +2377,12 @@ and fmt_expression c ?(box = true) ?(pro = noop) ?eol ?parens
         @@ wrap_if outer_parens (str "(") (str ")")
         @@ (fmt_longident txt $ Cmts.fmt_within c loc $ fmt_atrs)
   | Pexp_ifthenelse (if_branches, else_) ->
+      let parens = parens || has_attr in
+      let cmts_before = Cmts.fmt_before c ?eol pexp_loc in
+      let pro_outer, pro_inner =
+        if Params.Exp.ifthenelse_inner_pro ~parens ~ctx0 then (noop, pro)
```

Comment:
I think `pro_inner` is where the pro should be in every cases. This seems to create a regression with `--if-then-else=fit-or-vertical` but I think that option is the special case, not the context.

Reply:
It seemed to me that making less changes would be less likely to lead to regressions

Reply:
Anyway, I made the change